### PR TITLE
update pull request template

### PR DESCRIPTION
**Pull request by cjerad**
**Issue, if available:**

**Description of changes:**
Add reminders to the PR template to update docs on https://karpenter.sh.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by geoffcline — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
-**Issue, if available:**
+**1. Issue, if available:**
 
 
-**Description of changes:**
+**2. Description of changes:**
 
 
+**3. Do the changes impact the accuracy of information at https://karpenter.sh -- especially the [Getting Started Guide](https://karpenter.sh/docs/getting-started) or [Development Guide](https://karpenter.sh/docs/development-guide)?** YES/NO
+* **3.1. If yes, does this PR include updates?** YES/NO
```
Comment (ellistarn) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: Can can collapse these into: `Related docs issue (or N/A):`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
 
-**Description of changes:**
+**2. Description of changes:**
 
 
+**3. Does this change impact docs?**
+- [ ] Yes, PR includes docs updates
+- [ ] Yes, issue opened: *link to issue*
+- [ ] No
+
```
Comment (geoffcline) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: 💯 🥇

**Review by geoffcline — APPROVED** [review]
strongly approve

**Review by bwagner5 — APPROVED** [review]
energetically approve ⚡️

**Comment by ellistarn** [comment]
merge'd

**Pull request merged**